cmake/external/gtest.cmake

```diff
@@ -34,9 +34,15 @@ IF(WITH_TESTING)
             "${GTEST_INSTALL_DIR}/lib/libgtest_main.a" CACHE FILEPATH "gtest main libraries." FORCE)
     ENDIF(WIN32)
 
+    IF(WITH_MKL_LITE)
+        # wait for mkl downloading completed
+        SET(GTEST_DEPENDS   ${MKL_LITE_PROJECT})
+    ENDIF()
+
     ExternalProject_Add(
         extern_gtest
         ${EXTERNAL_PROJECT_LOG_ARGS}
+        DEPENDS         ${GTEST_DEPENDS}
```

Comment:
Not really depends, but if not wait for the download will get error can not find liomp5.